### PR TITLE
Merge bbox-polygon and bbox, center, polyline decode

### PR DESCRIPTION
**Pull request by armantorkzaban**
Takes the features from PRs: bbox-polygon from #98 and bbox, center and polyline decode from #96

### CODE REVIEW

File: lib/src/geojson.dart [hunk 1]
```diff
@@ -317,12 +321,30 @@ class BBox extends CoordinateType {
...
+    num? lng1,
+    num? lat1,
+    num? lng2,
+    num? lat2,
+  }) =>
+      BBox(
+        lng1 ?? this.lng1,
+        lat1 ?? this.lat1,
+        lng2 ?? this.lng2,
+        lat2 ?? this.lat2,
```
Comment (lukas-h) [inline lib/src/geojson.dart:345]: what about the altitudes alt1, alt2?

**Review by lukas-h — APPROVED** [review]

File: lib/src/geojson.dart (hunk 1 above)
Comment (lukas-h) [reply lib/src/geojson.dart:345]: can be fixed later. go ahead and merge!

File: README.md [hunk 2]
```diff
@@ -76,9 +76,9 @@ Any new benchmarks must be named `*_benchmark.dart` and reside in the
 - [ ] along
 - [ ] area
 - [ ] bbox
```
Comment (lukas-h) [inline README.md:78]: Ok. just found after merging, that bbox is not mentioned here

Comment (armantorkzaban) [reply README.md:78]: done!

File: lib/src/geojson.dart (hunk 1 above)
Comment (armantorkzaban) [reply lib/src/geojson.dart:345]: I added those as well. But I assume that those should be done in other places too.

Comment (lukas-h) [reply lib/src/geojson.dart:345]: But you did it after merge?

Comment (armantorkzaban) [reply lib/src/geojson.dart:345]: Yes, after merging. 
We need a PR

**Pull request merged**